examples/cancellablecontainers.jl

```diff
@@ -114,8 +114,11 @@ end
 
 # ## Generic cancellable operations
 #
-# It is often useful to cancel blocking operation safely. It can be expressed by
-# reagents quite naturally.
+# It is often useful to cancel a blocking operation safely. This can be
+# expressed quite naturally using reagents — we'll create a CancellationToken
+# which can be composed with any blocking operation. The pattern is similar to
+# the golang idiom of listening to a `Done` channel as one of the blocking
+# operations in a `select` statement.
```

Comment:
Rob Pike does not like Go to be called "golang." Let me pay some respect to him here :)

```suggestion
# [the Go idiom of listening to a `Done` channel](https://pkg.go.dev/context)
# as one of the blocking operations in a `select` statement.
```
j